### ISSUE TITLE
305063867 Create Gap but not available for Other Transcript (Team)

### ISSUE THREAD

**Issue by FreeREGcomputer** [FreeUKGen/FreeUKRegProductIssues#1533]
Issue reported by **alycymru** at 2024-08-31 19:57:47 UTC
Time: 2024-08-31T19:53:10+00:00
Session ID: 26a0f452ee95ce14e5b3beca87897e20
Problem Page URL: [/registers/596ccfddf493fd31bfe2422a?locale=en](/registers/596ccfddf493fd31bfe2422a?locale=en)
Previous Page URL: [https://www.freereg.org.uk/manage_counties/place_range?locale=en](https://www.freereg.org.uk/manage_counties/place_range?locale=en)
Reported Issue:
I need to create a gap for a chapel but there is no button for Create Gap available when the register type has been used. This issue might be present for other register types (e.g. Memorial Inscriptions, Phillimores)as well. Parish registers are not affected. Can we please get this button available in OT at least?
https://www.freereg.org.uk/uploads/feedback/screenshot/66d375bbb18f7ce2460a4cdb/No_create_cap_.jpg    

![Screenshot](https://www.freereg.org.uk)

**Comment by stsccfr** [comment]
(Firstly, I should say that neither of the two links provided in the original post are working.  The link to freereg.org.uk leads to a page that says "Missing Page. You have requested a page that does not exist on our server. Please make sure that you have actually accessed the [FreeReg Website](http://www.freereg.org.uk/)".  The other link for the screenshot is supposed to go to camo.githubusercontent.com but gives an error saying "Non-Image content-type returned".  If we ourselves can fix this, that would be worth doing because screenshots can be very helpful.)

I used test3 to create an OT (Other Transcript) register for Yew Tree, which is the Place Name that we are using for FreeComETT testing.  After doing this I was given an action button to Create Gap, and I used it to create a gap of 1642-1660 for burials.  The interface for creating a gap includes a field for providing a Reason, but that field was not usable.  The mouse cursor can't be put inside the text box to type a reason.  Despite this, I clicked Submit and the system told me that "Addition of Gap was successful".  So this is partly working, but we need to fix the field for entering a Reason for the gap.

Assigning back to Vino.

**Comment by DeniseColbert** [comment]
Dale to test on Live

**Comment by stsccfr** [comment]
Yes, this is working now on live.  I verified that the Create Gap button appears for all register types (Other Transcript, Other Register, Bishop's Transcript, Monumental Inscriptions, Parish Register, Archdeacon's Transcript, and Phillimore's Transcript) and clicking the button takes me to the page for creating a date gap.  I think this can be closed.
